packages/email-petition/src/email-petition.tsx

```diff
@@ -50,6 +53,22 @@ export function EmailPetition(props: {
   onSubmit?: () => void;
   debug: boolean;
   test: boolean;
+  /**
+   * City selection mode prop:
+   * - "sonomaCountyDropdown": Uses static dropdown with Sonoma County cities (no API calls)
+   * - "autocompleteTextbox": Uses Google Geocoding API for zip to city lookup and Google Places API (New) for city autocomplete
+   *
+   * If using "autocompleteTextbox", you must provide:
+   * - areaScope prop for geographic validation
+   * - NEXT_PUBLIC_GOOGLE_MAPS_PLACES_NEW_API_KEY for frontend autocomplete in service/.env.local
+   * - GOOGLE_MAPS_GEOCODING_API_KEY for backend ZIP lookup in [website]/.env
```

Comment:
Please remove this line, since it's about the backend rather than frontend, so it may get out of date (also, "[website]/.env" is inaccurate). Developers should read/set the server configuration when they debug for the first time and otherwise see a helpful error message if they forget to set it.